airflow/models/taskinstancehistory.py

```diff
@@ -58,7 +59,12 @@ class TaskInstanceHistory(Base):
     """
 
     __tablename__ = "task_instance_history"
-    id = Column(Integer(), primary_key=True, autoincrement=True)
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    task_instance_id = Column(
+        String(36).with_variant(postgresql.UUID(as_uuid=False), "postgresql"),
+        nullable=False,
+    )
+    try_id = Column(UUIDType(binary=False), nullable=False, unique=True)
```

Comment:
I don't like this on the history table - the try_id seems like the perfect primary key for this table

Reply:
Updated it as the primary key